eng/targets/Helix.Common.props

```diff
@@ -12,6 +12,8 @@
     <HelixAvailablePlatform Include="Linux" />
 
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.ClientRS4.VS2017.Open" Platform="Windows" />
+    <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
+    <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
```

Comment:
Are the agents in these queues actually running the named OS?

Reply:
They run the Windows Server equivalent, but per discussion with Eilon, we think this is sufficient for downlevel Windows testing.